test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2236,6 +2236,146 @@ Describe "Invoke-WebRequest tests" -Tags "Feature", "RequireAdminOnWindows" {
             $pathologicalRatio | Should -BeGreaterThan 5
         }
     }
+
+    Context 'Invoke-WebSession: Connection persistence in a WebSession' {
+        $connWarning = [regex]::new('WebSession.+HTTP')
```

Comment:
I'd add a comment with resource string name and its content for what message we expect.

Reply:
Done. Also updated variable name to better match the resource string.